diesel/src/migrations/mod.rs

```diff
@@ -264,6 +263,7 @@ pub fn search_for_migrations_directory(path: &Path) -> Result<PathBuf, Migration
 }
 
 #[cfg(test)]
+#[cfg_attr(feature = "clippy", allow(result_unwrap_used))]
```

Comment:
Isn't this redundant with https://github.com/diesel-rs/diesel/pull/305/files#diff-7ab3c70a207718b2fce878a8f925a415R22?